### PR TITLE
Minor changes to file.go

### PR DESCRIPTION
**Pull request by LipuFei**
- Use File.WriteString() instead of io.WriteString()
- Log the real written count

### CODE REVIEW

File: file.go [hunk 1]
```diff
@@ -20,11 +19,11 @@ func CreateFileWith(filename, contents string) {
 		Fatalln("CreateFileWith failed:", err)
 	}
 	defer f.Close()
-	_, err = io.WriteString(f, contents)
+	n, err := f.WriteString(contents)
 	if err != nil {
 		Fatalln("CreateFileWith failed:", err)
 	}
-	log.Printf("written %d bytes to %s\n", len(contents), filename) // show absolute name
+	log.Printf("written %d bytes to %s\n", n, filename) // show absolute name
```
Comment (emicklei) [inline file.go]: why is n different from len(contents)?

Comment (LipuFei) [inline file.go]: WriteString() returns the number of characters that have been actually written, which can be different from what we give.

Comment (emicklei) [inline file.go]: so we should log them both!

**Comment by LipuFei** [comment]
@emicklei Now it logs both counts.

**Pull request merged**